borg/testsuite/helpers.py

```diff
@@ -104,6 +104,36 @@ def test(self):
         self.assert_equal(self.evaluate(['/etc/', '/var'], ['dmesg']),
                           ['/etc/passwd', '/etc/hosts', '/var/log/messages', '/var/log/dmesg'])
 
+class OSXPatternNormalizationTestCase(BaseTestCase):
```

Comment:
If we use `sys.platform in ('darwin', )`, they are, so we can put them into a `skipUnless` clause ...

````python
@unittest.skipUnless(sys.platform.startswith('darwin'), 'OS X only test')
class NormalizationTestCase(BaseTestCase):
    def testComposedUnicode(self):
        pattern = 'b\N{LATIN SMALL LETTER A WITH ACUTE}'
        ...
````